sdk/src/compute_budget.rs

```diff
@@ -41,6 +41,8 @@ pub enum ComputeBudgetInstruction {
     /// Set a compute unit price in "micro-lamports" to pay a higher transaction
     /// fee for higher transaction prioritization.
     SetComputeUnitPrice(u64),
+    /// Set a specific transaction-wide account data size limit, in bytes, is allowed to load.
+    SetLoadedAccountsDataSizeLimit(u32),
```

Comment:
:sob:

Reply:
sad because no TLV here? I bounced around the idea with multiple parties, as I wasn't fully convinced that mixed use of TLV and whatever-we-have will be satisfying. The consensus is mainly on "less programs, more instructions per program". So i decided to keep this program consistent in style, leave TLV to next new program (say ApplicationFee :D )

Reply:
> TLV means what?

i had proposed that we stop adding new compute budget instructions for each requestable parameter and instead add one final compute budget instruction that takes as data a TLV structure similar to [what token22 uses for extensions](https://github.com/solana-labs/solana-program-library/blob/master/token/program-2022/src/extension/mod.rs).  idea here being that the instructions add both space and compute overhead to transactions and their process.  also, in a future transaction format version, the TLV structure could be made a first-class citizen, eliminating the need for compute budget instructions and the compute budget program at all